### PR TITLE
[Component with its own class name as service type] LPS-199148 Refactor BatchEngineImportStrategyFactory and ResourceBundleLoaderProvider component

### PR DESCRIPTION
**Pull request by Valmir-unicap**

### CODE REVIEW

**Comment by Valmir-unicap** [comment]
ci:test:sf

File: modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/util/ResourceBundleLoaderProviderUtil.java [hunk 1]
```diff
@@ -37,17 +34,16 @@ public ResourceBundleLoader getResourceBundleLoader(
 			ResourceBundleLoaderUtil.getPortalResourceBundleLoader());
 	}
 
-	@Activate
-	protected void activate(BundleContext bundleContext) {
-		_serviceTrackerMap = ServiceTrackerMapFactory.openSingleValueMap(
-			bundleContext, ResourceBundleLoader.class, "bundle.symbolic.name");
-	}
+	private static final ServiceTrackerMap<String, ResourceBundleLoader>
```
Comment (rafaprax) [inline modules/apps/configuration-admin/configuration-admin-web/src/main/java/com/liferay/configuration/admin/web/internal/util/ResourceBundleLoaderProviderUtil.java:37]: Hey @Valmir-unicap, you do not need this tracker. Use the `ResourceBundleLoaderUtil` instead. Thx

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/template/TemplateHandlerRegistryImpl.java [hunk 2]
```diff
@@ -153,6 +155,7 @@ protected void deactivate() {
 
 	private final Map<TemplateHandler, ServiceRegistration<?>>
 		_serviceRegistrations = new ConcurrentHashMap<>();
+	private ServiceTrackerMap<String, ResourceBundleLoader> _serviceTrackerMap;
```
Comment (rafaprax) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/template/TemplateHandlerRegistryImpl.java:158]: Hey @Valmir-unicap, you do not need this tracker. Use the `ResourceBundleLoaderUtil` instead. Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by Valmir-unicap** [comment]
Fix finish!!! Sent to team repository.
Pr: https://github.com/liferay-core-infra/liferay-portal/pull/5928

**Pull request closed without merging**